lib/install/is-dev.js

```diff
@@ -2,6 +2,26 @@
 var isDev = exports.isDev = function (node) {
   return node.package._requiredBy.some(function (req) { return req === '#DEV:/' })
 }
-exports.isOnlyDev = function (node) {
+var isOnlyDev = exports.isOnlyDev = function (node) {
```

Comment:
@terinjokes The `is-extraneous` use sure looks needed to me. Without it, all dev-only deps would be listed as extraneous.